packages/design-system-old/src/AppIcon/index.tsx

```diff
@@ -244,6 +244,7 @@ const YenIcon = importSvg(() =>
 const SteamBowlIcon = importSvg(() =>
   import("../assets/icons/ads/app-icons/steam-bowl.svg"),
 );
+const PackageIcon = importSvg(() => import("../assets/icons/ads/package.svg"));
```

Comment:
why is this new icon being added to design system old? If it is because of the usage of app icon, can we consider replacing it with icon from ADS 2.0?

Reply:
@albinAppsmith The icon already existed on design-system-old. I simply added its usage in AppIcon. It would be difficult to use Icon instead of AppIcon because application cards needs to use AppIcon I thin. Let me know your thoughts.
CC: @ashit-rath